### PR TITLE
Reworks Bag of Holding explosions.

### PR DESCRIPTION
**Pull request by BurgerLUA**
## About The Pull Request

Bag of holding explosions now have reduced range (6 to 4).
Bag of holding explosions no longer gib the user.
There is no longer a singularity inside the bag of holding area.

## Why It's Good For The Game

Given that this is an RP server without much in the way of suicide objectives or kill objectives, the Bag of Holding bomb is seldom used except in rare cases where someone comments on the github here about how they used it in a hostage situation a month a go or something.

This should make them used slightly more but the effects of using it aren't as bad to the victims and the people around.

## Changelog
:cl: BurgerBB
balance: Reduced the range of bag of holding bombs from 8 to 6. Bag of Holding bombs no longer hardgib the user. Bag of Holding bombs no longer have a singularity in the center.
/:cl:

### CODE REVIEW

**Comment by improvedname** [comment]
Might aswell remove the feature if its like this

**Comment by jjpark-kb** [comment]
> Might aswell remove the feature if its like this

This is sort of not helpful...
This is a good change. It makes them less lethal, but still have the same effect.

**Comment by improvedname** [comment]
Yeah thats fair but te singularity in the middle was mostly to punish the BoH bomber

**Comment by BurgerLUA** [comment]
> Yeah thats fair but te singularity in the middle was mostly to punish the BoH bomber

BOH bombers were insta-gibbed every time previously. Your actual placement on the tiles is actually random.

File: code/datums/components/storage/concrete/bag_of_holding.dm [hunk 1]
```diff
@@ -18,8 +18,7 @@
 		to_chat(user, "<span class='danger'>The Bluespace interfaces of the two devices catastrophically malfunction!</span>")
 		qdel(W)
 		playsound(loccheck,'sound/effects/supermatter.ogg', 200, 1)
```
Comment (Trilbyspaceclone) [inline code/datums/components/storage/concrete/bag_of_holding.dm:20]:
```suggestion
		playsound(loccheck,'sound/effects/supermatter.ogg', 200, 1)
		//user.gib(TRUE, TRUE, TRUE)
		for(var/turf/T in range(4,loccheck)) //Skyrat changes, lowers range from 6 to 4 removes selfgib
```

**Comment by Akrilla** [comment]
Can merge after conflict

**Pull request merged**